### PR TITLE
Amnesia 

### PR DESCRIPTION
**Pull request by Jsfrulle**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  <title>Amnesia Ibiza</title>
+  </head>
+  <body>
+
+<div class="container"> 
+
+
+
+
+<header class="header-block">
```
Comment (Rephili) [inline index.html:24]: Really nice header with the video and audio

File: index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+let accordions = document.querySelectorAll("div.accordion");
+
+for(let i = 0; i < accordions.length; i++){
```
Comment (Rephili) [inline index.js:4]: Good job!

File: style.css [hunk 3]
```diff
@@ -0,0 +1,640 @@
...
+.header-block__audio {
+  position: absolute;
+  width: 25vw;
+  margin-left: 75vw;
+  align-self: baseline;
+  border: solid 0.1px rgb(237, 243, 243);
+  background-color: rgb(237, 243, 243);
+}
+
+.header-block__video {
```
Comment (Rephili) [inline style.css:37]: Cool! I will try to implement this in future projects (haven't tried using Z-index before). :)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,640 @@
...
+  outline: none;
+  border: 3px dotted rgb(100, 100, 102);
+  color: rgb(22, 56, 52);
+  font-weight: bolder;
+}
+
+
+/* Checkbox - did try a diffrent way to style these (used a other method on the radio-btn)*/
+
+input[type="checkbox"] {
```
Comment (Rephili) [inline style.css:230]: I learn a lot while reading your code. :) I didn't have time to style my checkbox, but now I see how you solved it, thanks!

File: style.css [hunk 5]
```diff
@@ -0,0 +1,640 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.signup-form {
+  display: block;
+  overflow: hidden;
+  margin: 1vw 10vw;
+  backdrop-filter: blur(10px);
```
Comment (Rephili) [inline style.css:179]: I like the blur effect!

File: style.css [hunk 6]
```diff
@@ -0,0 +1,640 @@
...
+
+
+
+
+/* MEDIA QUERIES */
+
+
+/*  PHONE  */
+
+@media (min-width: 0) and (max-width: 767px) {
```
Comment (Rephili) [inline style.css:411]: You may consider making the sign-up form and the accordion a little bit bigger in mobile version in order to make it more readable. :)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+
+  </header>
+  
+
+
+<section class="form-div">
+   <div class="signup-form"> 
+
+    <!-- Signup form -->
+    <form class="form-text" action="https://httpbin.org/anything" method="post">
```
Comment (Rephili) [inline index.html:52]: I submitted the form and it works

File: index.html [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+      <div class="answer">
+        <p>If tickets aren’t sold out before the festival begins, 
+          you can buy them at the entrance at a slightly higher price.</p>
+      </div>
+    
+    </section>
+
+ 
+
+<footer class="footer-div">
```
Comment (Rephili) [inline index.html:178]: Your code is great but you may have a look at the structure of it to make it even more easy to read. :) In VS code I think there is a shortcut to format it (Shift+Option+F on a Mac).

**Review by Rephili — COMMENTED** [review]
Great project, Jessica! I learned some more advanced CSS while doing this code review and I will try to implement some of that in my next projects. :) I wrote some more single comments in the code.

**Comment by Jsfrulle** [comment]
Hi Rebecca, I fixed the code structure and size of the form and the accordion. I'm glad to hear that you get some inspiration for a future project! Thanks for the good and constructive feedback! :)

**Pull request closed without merging**